.travis.yml

```diff
@@ -67,11 +67,15 @@ deploy:
   skip_cleanup: true
   file_glob: true
   file:
-  - _out/cmd/cdi-controller/cdi-controller
-  - _out/cmd/cdi-importer/cdi-importer
-  - _out/cmd/cdi-cloner/cloner_startup.sh
-  - manifests/generated/cdi-controller.yaml
-  - manifests/generated/cdi-controller.yaml.j2
+  - _out/cmd/cdi-controller/*
+  - _out/cmd/cdi-importer/*
+  - _out/cmd/cdi-apiserver/*
+  - _out/cmd/cdi-cloner/*
+  - _out/cmd/cdi-operator/*
+  - _out/cmd/cdi-uploadproxy/*
+  - _out/cmd/cdi-uploadserver/*
+  - manifests/generated/*.yaml
```

Comment:
Not sure the functional testing manifests should be included (file-host.yaml and registry-host.yaml).  Also wonder if we should ship cdi-controller.yaml.  Maybe deprecate it now to be removed later?  If we do include it, we should probably  add creation of the 'cdi' namespace to the manifest.

Reply:
Well I would like to avoid having to manually add each new file we create to this list. Because someone will forget to add it at some point. When/if we remove cdi-controller.yaml, doing it like this will automatically remove it from the release as well.